### DISCUSSION TITLE
Add the option to adjust print speed/set max print speed?

### DISCUSSION THREAD

**Discussion by RhythmBlade** [OrcaSlicer/OrcaSlicer#3543] [Ideas]
I have tried printing with orca recently and have just been getting fail after fail after fail strictly because the printer is moving too fast. Im trying to learn all the movement speeds and what each do but to someone whos learning its a bit overwhelming/intimidating to see all the options you have to potentially change. Adding a feature that allows you to set a max or custom print speed seems like a pretty good quality of life thing for new and current users. Prusaslicer and Cura both have this option. (I print on a CR6-SE for reference)

**Comment by HVR88** [comment]
Default print speeds for the CR-6SE in Orca are slow.

Using the same printer and using the original glass bed, I've had only perfect prints with super smooth exterior walls at every setting from standard (.20) and better. Even setting Super Draft (.28) still keeps very low speeds. 

If you're having issues I don't see it related to speed unless you have a custom profile that wasn't downloaded as part of Orca.

**Reply by VK1IAN** [reply]
I think this would be a good feature; perhaps in the "filament" settings.
I am currently using a Qidi Q2, but the PLA I have been given has a printing speed of 10 - 60 mm/s (from the manufacturer's specs on the reel).
So, I need to cap the speed at 60mm/s.
The only way I can see to this currently (2.3.1-beta) is to adjust ALL the speed settings in Process->Speed tab.
More than happy to be corrected if this is not the case.
